src/python/pants/backend/python/lint/pylint/rules.py

```diff
@@ -131,6 +131,10 @@ async def pylint_lint_partition(partition: PylintPartition, pylint: Pylint) -> L
         PexFromTargetsRequest,
         PexFromTargetsRequest.for_requirements(
             (field_set.address for field_set in partition.field_sets),
+            # NB: These constraints must be identical to the other PEXes. Otherwise, we risk using
```

Comment:
Consider being specific about the other two pexes, in case that changes in the future, e.g.
```suggestion
            # NB: These constraints must be identical to the pylint and pylint runner PEXes. Otherwise, we risk using
```

Reply:
Even if we add other PEXes, those need to use the same constraints too. Everything merged via `--pex-path` needs it.

Reply:
Ok, that makes sense!